lib/transport/mailgun.js

```diff
@@ -3,16 +3,18 @@ var request = require('request');
 
 var MAILGUN_MESSAGES_URL = 'https://api.mailgun.net/v3/' + config.get('transport.mailgun.domain') + '/messages';
 var MAILGUN_PASS = config.get('transport.mailgun.apiKey');
+var log = require('../util/logger');
 
 function sendEmails(emails, content, callback) {
 
   var data = {
-    to: emails,
+    to: emails || [content.to],
```

Comment:
this optional `[content.to]` is not necessary, every call to sendEmails should receive emails on the first parameter.